src/app/ui/document_view.cpp

```diff
@@ -160,6 +160,11 @@ class PreviewEditor : public Editor,
     setCustomizationDelegate(this);
   }
 
+  ~PreviewEditor()
+  {
+	setCustomizationDelegate(NULL);
```

Comment:
It looks like this line is indented with a tab char, we use 2 spaces for indentation.